services/ymax-planner/scripts/process-tx-lib.ts

```diff
@@ -204,7 +204,8 @@ export const processTx = async (
       );
 
       // Process the transaction with lookback mode
-      await handlePendingTx(tx, txPowers, timestampMs);
+      const abortController = new AbortController();
+      await handlePendingTx(tx, txPowers, timestampMs, abortController.signal);
```

Comment:
An AbortSignal input should always be optional, as it is in web APIs.
```suggestion
      await handlePendingTx(tx, txPowers, timestampMs);
```